### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by Shou** [muammar/mkchromecast#244]
Looks like there've been quite a few fixes since the last release in December 2017, I'd love to have `mkchromecast` available in package managers with those fixes.

**Comment by Aimmig** [comment]
Yeah would be really great, escpecially since the yt-download is still broken in the last release. Meanwhile using the github version, which works perfectly.

**Comment by bobvanderlinden** [comment]
I'm trying to package mkchromecast for NixOS. It would be really helpful to have a release, so that the package can have a proper version number. Creating a tag from master is fine by me.

**Comment by Shou** [comment]
@bobvanderlinden https://github.com/NixOS/nixpkgs/pull/57033

Looks like we're here for the same reason 😁

**Comment by bobvanderlinden** [comment]
@Shou Haha, that's quite a coincidence. Quite a nice package. I've closed mine for yours. Cherry on top would indeed be a new version.

**Comment by bjornfor** [comment]
@muammar: what is missing before a release can be tagged?

**Comment by muammar** [comment]
> @muammar: what is missing before a release can be tagged?

The macOS app. Whatever recipe I had to build it is not working anymore. If someone could help on that would be great.

**Comment by pasikarkkainen** [comment]
Any developments related to getting a new mkchromecast release out? Is the macOS build still a blocker?

**Comment by austinbutler** [comment]
Is there a related issue for the macOS problem?

**Comment by DJCrashdummy** [comment]
although this is the older ticket, i would consider this as the duplicate of #348 becaues it contains a liitle bit more information about a possible solution.
